recipes/tlsh/meta.yaml

```
@@ -0,0 +1,108 @@
+{% set name = "tlsh" %}
+{% set version = "3.11.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/trendmicro/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 09e116130d98d6ed180624ff9f7da22b669ccd6f24e316f59019bf56436ee444
+  patches:
+    - make.patch
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - make
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  commands:
+    - echo tlsh metapackage has no tests
+
+outputs:
+  - name: libtlsh
+    script: install-libtlsh.sh
+    build:
+      run_exports:
+        - {{ pin_subpackage('libtlsh', max_pin='x.x') }}
+    requirements:
+      build:
+        - {{ compiler('c') }}
+        - {{ compiler('cxx') }}
+        - cmake
+        - make
+    test:
+      commands:
+        - echo libtlsh tested are during build
```

Can you put a file existence test here like:
```suggestion
        - test -f ${CONDA_PREFIX}/lib/libtlsh.${SHLIB_EXT}
```
(I suggest it as I’ve accidentally built empty packages before when only echoing)

---

Added, thanks!